src/main/java/ca/uwaterloo/swag/pilaipidi/PilaiPidi.java

```diff
@@ -167,6 +168,23 @@ private static void loadFunctiosFromXmlInputStream(List<String> functionList, In
         }
     }
 
+    private static void loadFunctionsWithParamsFromXmlInputStream(HashMap<String, Integer> functionList,
+                                                                  InputStream xmlInputStream) throws ParserConfigurationException, IOException, SAXException {
```

Comment:
Can you check whether this line is over 120 col-space?

Reply:
Fixed.